### DISCUSSION TITLE
Page not found Error

### DISCUSSION THREAD

**Discussion by pardhav** [facebook/docusaurus#5192] [Q&A] [answered]
Hi All,

My Docs folder has multiple sub-folders, with multiple .mdx files. Every time I try to click on the Docs tab I get a page not found error. Is there anything wrong with the setup? I'm using Docusaurus classic template. 

Below is my docs folder:

![image](https://user-images.githubusercontent.com/17379426/126329981-20ced9f7-43fa-4615-b35a-fde00708a5dd.png)

Here is my navbar config
```js
navbar: {
      title: "Apex Docs",
      logo: {
        alt: "Apex Docs",
        src: "img/logo.svg",
      },
      items: [
        {
          position: "left",
          label: "Docs",
          to: "/docs",
        },
        { to: "/blog", label: "Blog", position: "left" },
        // {
        //   href: "https://github.com/facebook/docusaurus",
        //   label: "GitHub",
        //   position: "right",
        // },
      ],
    }
```

**Comment by slorber** [comment, marked as answer]
You must use `slug: /` on one of your docs, so that it is hosted at the root `/docs/` path instead of `/docs/myDoc`

The "docs tab" (as you call it) is not a special navbar item, it's just a regular navbar link,  so you are responsible for making the link target an existing page. In your case, no page exists at `/docs` path so this leads to a 404. Make sure `/docs` page exist before linking to it.

**Reply by pardhav** [reply]
I was not aware of the ```slug```. Thanks for the help!
